src/apps/activities/domain/response_values.py

```diff
@@ -37,6 +39,22 @@
 from apps.shared.domain import PublicModel, validate_color, validate_image, validate_uuid
 
 
+class PhrasalTemplateFieldType(str, Enum):
+    SENTENCE = "sentence"
+    ITEM_RESPONSE = "item_response"
+    LINE_BREAK = "line_break"
+
+
+class PhrasalTemplateDisplayMode(str, Enum):
+    BULLET_LIST = "bullet_list"
+    SENTENCE = "sentence"
+    ITEM_INDEX = "item_index"
+    BULLET_LIST_OPTIONS_TEXT = "bullet_list_option_text"
+    BULLET_LIST_TEXT_OPTIONS = "bullet_list_text_option"
+    SENTENCE_OPTION_TEXT = "sentence_option_text"
+    SENTENCE_TEXT_OPTION = "sentence_text_option"
```

Comment:
I think the names of these display modes could be updated to better reflect the design that inspired it. Totally optional though
```suggestion
    BULLET_LIST_OPTION_ROW = "bullet_list_option_row"
    BULLET_LIST_ROW_OPTION = "bullet_list_row_option"
    SENTENCE_OPTION_TEXT = "sentence_option_row"
    SENTENCE_TEXT_OPTION = "sentence_row_option"
```

I think it would also be good to replace `option` with `column` since that's what's actually being referenced, but either one will work

Reply:
done!